src/AWS.Deploy.CLI/ConsoleUtilities.cs

```diff
@@ -307,6 +308,58 @@ public string AskUserForValue(string message, string defaultValue, bool allowEmp
             return userValue;
         }
 
+        public SortedSet<string> AskUserForList(SortedSet<string> listValues)

```

Comment:
When going through the experience of adding a VPC connector I added a subnet and security group. I wanted to add a second subnet to my VPC connector which goes through all of the settings including both subnets and security groups. This was awkward because I didn't want to change my security group but I was only given the option of add/update/delete security groups.

We need some way of saying I don't want to change a list when going through the settings.

Reply:
Right now, if you choose to Add a new value and leave it empty, it gives the behavior you are looking for. Though i need a more explicit way to do that. I'll look into it